conf/modules.config

```diff
@@ -651,6 +651,23 @@ process {
         ]
     }
 
+    withName: KMCP_SEARCH {
+        publishDir = [
+            path: { "${params.outdir}/kmcp/${meta.db_name}/" },
+            mode: params.publish_dir_mode
+        ]
+    }
```

Comment:
I think since `KMCP_SEARCH` is necessary in order to perform `KMCP_PROFILE`, it should not be optional. 
I will update the PR with a prefix and I will double-check any extra parameters.

Reply:
Sorry I mean saving the output

Reply:
Yes, it can be optional.